src/Makefile.sources

```diff
@@ -188,6 +188,7 @@ HB_OT_headers = \
 	hb-ot-name.h \
 	hb-ot-shape.h \
 	hb-ot-var.h \
+	hb-aat.h \
```

Comment:
WHATS THIS!??!?!?!!!

Reply:
Hmm?

Reply:
Err. Sorry not this. You included it in hb-ot.h? That I meant to object to.

Reply:
Done